source/header_cell.ts

```diff
@@ -0,0 +1,105 @@
+import {Sorting} from './sorting';
+import {Filter} from './filter';
+
+/** Header of a column of a table. */
+export class HeaderCell {
+
+  /** Constructs a new HeaderCell.
+    * @param name - The name to display if there is enough available space.
+    * @param shortName - The name to display if there is not enough space.
+    * @param isSortable - Indicates if the column is sortable.
+    * @param sorting - The current sorting of the column.
+    */
+  constructor(name: string, shortName: string, isSortable: boolean = true,
```

Comment:
Avoid fields that have the potential to introduce inconsistencies, create APIs that are correct by construction. ```isSortable``` is inconsistent with ```Sorting```.